pkg/engine/variables/operator/allin.go

```diff
@@ -57,7 +57,7 @@ func allKeyExistsInArray(key string, value interface{}, log logr.Logger) (invali
 	switch valuesAvailable := value.(type) {
 	case []interface{}:
 		for _, val := range valuesAvailable {
-			if wildcard.Match(key, fmt.Sprint(val)) {
+			if wildcard.Match(fmt.Sprint(val), key) || wildcard.Match(key, fmt.Sprint(val)) {
```

Comment:
Does it really fix the issue ?
Can you add a unit test please ?

Reply:
Sure, I think so.

Reply:
I have add the ut.